hash_practice/exercises.py

```diff
@@ -1,19 +1,57 @@
+# Input: ["eat", "tea", "tan", "ate", "nat", "bat"],
+# Output:
+# [
+#   ["ate","eat","tea"],
+#   ["nat","tan"],
+#   ["bat"]
+# ]

```

Comment:
As a style thing, you'll usually want to remove comments like this that are not really related to explaining why the code is designed a certain way.